source/guide_invoiceplane.rst

```diff
@@ -0,0 +1,117 @@
+.. highlight:: console
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: lang-php
+.. tag:: web
+.. tag:: business
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/invoiceplane.png
+      :align: center
+
+############
+InvoicePlane
+############
+
+.. tag_list::
+
+InvoicePlane_ is a self-hosted open source application for managing your quotes, invoices, clients and payments.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`PHP <lang-php>`
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+We’re using :manual:`PHP <lang-php>` in the stable version 7.2. Since new Uberspaces are currently setup with PHP 7.1 by default you need to set this version manually:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version use php 7.2
+ Selected PHP version 7.2
+ The new configuration is adapted immediately. Patch updates will be applied automatically.
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+To install InvoicePlane we download the current version from the official website. ``cd`` to your :manual:`DocumentRoot <web-documentroot>` so the zip file will be under your ``html``.
+
+.. code-block:: console
+ :emphasize-lines: 2
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ wget https://www.invoiceplane.org/download/v1.5.10
+ [isabell@stardust html]$ unzip v1.5.10
+ [isabell@stardust html]$ rm v1.5.10
+ [isabell@stardust html]$ 
+
+Configuration
+=============
+
+Create the database, copy the configuration template, rename the ``htaccess`` file and add your URL ``https://isabell.uber.space`` in the ``ipconfig.php`` file under ``IP_URL``.
+
+ [isabell@stardust html]$ mysql -e "CREATE DATABASE ${USER}_ip"
+ [isabell@stardust html]$ cp ipconfig.php.example ipconfig.php
+ [isabell@stardust html]$ mv htaccess .htaccess
+ [isabell@stardust html]$
+
+After the installation you need to open https://isabell.uber.space/index.php/setup in your browser to finish your setup.
```

Comment:
Opening the site always redirects to `http://127.0.0.1/...` for me, unless I set `IP_URL=https://isabell.uber.space`.

Reply:
That part is described in the configuration section.

Reply:
ooh, up there. sorry!